config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  #test_Fang
+  get 'projects/:id/progress' => 'projects#progress'
```

Comment:
So we can link to it easily in the future:
```suggestion
  get 'projects/:id/progress' => 'projects#progress', as: :project_progress
```